docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,427 @@
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+You can retrieve the keywords and the group name of the stickers.
+
+The following example shows how to retrieve the keyword list:
+
+```
+void
+keyword_foreach_cb(const char *keyword, void *user_data)
+{
+    /* Your code */
+}
+
+void
+retrieve_keyword_list(sticker_consumer_h sticker_consumer)
+{
+    int ret;
+
+    /* Retrieve all keywords */
+    ret = sticker_consumer_keyword_list_foreach_all(sticker_consumer, keyword_foreach_cb, NULL);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+<a name="get_data_handle"></a>
+### Getting Sticker Information
+
+To obtain sticker information, use `sticker_data_get__XXX()`. To avoid memory leaks, free the character array variable with `free()` when no longer needed.
+
+The following example shows how to get the sticker information from the Sticker data handle:
+
+```
+void
+keyword_foreach_cb(const char *keyword, void *user_data)
+{
+    /* Your code */
+}
+
+void
+get_sticker_data(sticker_data_h sticker_data)
+{
+    int ret;
+    char *app_id;
+    sticker_data_uri_type_e type;
+    char *uri;
+    char *group;
+    char *thumbnail;
+    char *description;
+    char *last_updated;
+
+    /* Gets the name of the sticker provider application */
+    ret = sticker_data_get_app_id(sticker_data, &app_id);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+
+    /* Gets the URI and URI type of the sticker */
+    ret = sticker_data_get_uri(sticker_data, &type, &uri);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+
+    /* Retrieves all keywords of the Sticker data handle */
+    ret = sticker_data_foreach_keyword(sticker_data, keyword_foreach_cb, NULL);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+
+    /* Gets the group name of the sticker */
+    ret = sticker_data_get_group_name(sticker_data, &group);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+
+    /* Gets the thumbnail local path of the sticker */
+    ret = sticker_data_get_thumbnail(sticker_data, &thumbnail);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+
+    /* Gets the description of the sticker */
+    ret = sticker_data_get_description(sticker_data, &description);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+
+    /* Gets the last updated date of the sticker */
+    ret = sticker_data_get_date(sticker_data, &last_updated);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+### Destroying Sticker Consumer Handle
+
+Destroy the Sticker consumer handle using `sticker_consumer_destroy()`:
+
+```
+void
+destroy_sticker_consumer_handle(sticker_consumer_h sticker_consumer)
+{
+    int ret;
+    ret = sticker_consumer_destroy(sticker_consumer);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+## Related Information
+-  Dependencies
```

Comment:
As mentioned in the earlier comment (https://github.com/Samsung/tizen-docs/pull/884#discussion_r336441466), please fix the indentation:

-  Dependencies
    - Tizen 5.5 and Higher for Mobile
    - Tizen 5.5 and Higher for Wearable

Reply:
Fixed.